README.md

```diff
@@ -169,6 +169,12 @@ Shade & Relocate:
 </build>
 ```
 
+## Compiling
+PaperLib is compiled using Gradle:
+```
+$ gradlew build
```

Comment:
Syntax differs between Linux and Windows.

Reply:
Not really? 
`gradlew build` works in CMD
`.\gradlew build` works in Powershell

Reply:
`./gradlew build` works everywhere. The existing line wouldn't work with Linux (which `$` implies) anyways, as it's not relative.